ark/type/parser/tuple.ts

```diff
@@ -414,8 +414,10 @@ export type validateInfixExpression<def extends InfixExpression, $, args> =
 			: def[1] extends ":" ? Predicate<type.infer.Out<def[0], $, args>>
 			: def[1] extends "=>" ? Morph<type.infer.Out<def[0], $, args>>
 			: def[1] extends "@" ? MetaSchema
-			: def[1] extends "=" ? type.infer.Out<def[0], $, args>
-			: validateDefinition<def[2], $, args>
+			: def[1] extends "=" ?
```

Comment:
So here, we want to make the check about whether it's primitive and then return the allowed forms